tests/test_unit_spec.py

```diff
@@ -207,7 +207,9 @@ def test_resolve_odxlinks(self) -> None:
             ecu_variant_patterns=[],
             comparam_spec_ref=None,
             prot_stack_snref=None,
-        )
+            diag_variables_raw=[],
+            variable_groups=NamedItemList(),
+            dyn_defined_spec=None)
```

Comment:
```suggestion
            dyn_defined_spec=None
        )
```

Reply:
I can remove the comma, but our coding style police (`yapf`) enforces the closing bracket to be on a separate line. (for many such list I like to use a comma for the last argument because it allows to add more items without having to modify the last line. This is currently inconsistent and just a IMO very unimportant detail..)